var/spack/repos/builtin/packages/onednn/package.py

```diff
@@ -85,12 +87,14 @@ class Onednn(CMakePackage):
         values=("ocl", "none"),
```

Comment:
Can you change `cpu_runtime`'s:
```python
("omp", "tbb", "seq")
```
to:
```python
(
    conditional("none", when="@2.3:"),
    "omp",
    "tbb",
    "seq",
    conditional("threadpool", when="@1.4:"),
    conditional("dpcpp", when="@2:"),
    conditional("sycl", when="@2:"),
)
```
and change `gpu_runtime`'s:
```python
values=("ocl", "none")
```
to:
```python
(
    "ocl",
    "none",
    conditional("dpcpp", when="@2:"),
    conditional("sycl", when="@2:"),
)
```
For future me, I found this using the following script:
```bash
fname=cmake/options.cmake

for tag in $(git tag -l | sort -Vr)
do
    echo
    echo $tag
    echo
    git checkout $tag $fname && grep -B1 'Unsupported .PU runtime' $fname && git restore --staged $fname && git restore $fname
done
```

Reply:
Done.